index.js

```diff
@@ -1,7 +1,59 @@
 // Iteration 1: Names and Input
+let hacker1 = "Ivan";
+let hacker2 = "Jose";
 
+console.log(`The driver's name is ${hacker1}`);
+console.log(`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
-
+if (hacker1.length > hacker2.length) {
+    console.log (`The driver has the longest name, it has ${hacker1.length} characters.`)
+  } else if (hacker2.length > hacker1.length){
+    console.log (`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+  } else {
+     console.log (`Wow, you both have equally long names, ${hacker1.length} characters!`)
+  }
 
 // Iteration 3: Loops
+// 3.1
+
+let result = "";
+hacker1 = `${hacker1}`;
```

Comment:
```suggestion
hacker1 = `${hacker1}`;
```
Esto en el fondo no hay que hacerlo, hacker1 en si ya es un string y ademas ya apunta a ese valor, asi que esto no te haria falta :D